Remove duplicate information in autoload page

skyace65 opened the pull request:
Removes duplicate info which made the page confusing to read. Closes #8312.

Piralein commented on tutorials/scripting/singletons_autoload.rst:
```diff
@@ -86,14 +87,6 @@ This means that any node can access a singleton named "PlayerVariables" with:
-If the **Enable** column is checked (which is the default), then the singleton can
-be accessed directly in GDScript, without requiring ``get_node()``:
-
-.. tabs::
- .. code-tab:: gdscript GDScript
-
-   PlayerVariables.health -= 10
-
```
Isn't this the recommended example, which should remain and the other removed?

Piralein commented on tutorials/scripting/singletons_autoload.rst:
```diff
@@ -73,7 +73,8 @@ in top-to-bottom order.
 .. image:: img/autoload_example.webp
 
-This means that any node can access a singleton named "PlayerVariables" with:
+If the **Enable** column is checked (which is the default), then the singleton can
+be accessed directly in GDScript, without requiring ``get_node()``:
```
If we remove the `get_node()` example, this could be rewritten to not mention the function at all.
Otherwise we should probably reverse it and add it as: "If you deactivate the `global variable` checkbox, the node can no longer be accessed by its name. Use the `get_node()` function to access it. + example". As the issue suggests.

skyace65 commented:
Yeah I probably shouldn't have made this PR while tired. Updated it. Switched which code example is deleted and removed the reference to ``get_node()``.

Piralein reviewed (approved):
The C# sections still mentions the `GetNode()` with typecast. Maybe the example should just be moved below the:

> Note that autoload objects (scripts and/or scenes) are accessed just like any other node in the scene tree

as it contains both the GDscript and C# example. Either way solves the linked issue.

mhilbrunner commented:
Thanks!

mhilbrunner commented:
Cherrypicked to 4.2.
